templates/repo/view_list.tmpl

```diff
@@ -50,8 +50,8 @@
 			<tr data-entryname="{{$entry.Name}}" data-ready="{{if $commit}}true{{else}}false{{end}}" class="{{if not $commit}}not{{end}}ready entry">
 				<td class="name four wide">
 					<span class="truncate">
+						{{svg (EntryIcon $entry)}}
```

Comment:
So we don't know if it's a submodule now?

Reply:
The entire icon logic (including this check) will be handled in the `EntryIcon` function